src/app/components/data/shared/file-group-picker/file-group-picker.component.ts

```diff
@@ -77,12 +77,26 @@ export class FileGroupPickerComponent implements ControlValueAccessor, OnInit, O
             this.fileGroupsData.params = {
                 storageAccountId,
             };
+
             this.fileGroupsData.setOptions({
                 filter: FilterBuilder.prop("name").startswith(Constants.ncjFileGroupPrefix),
             });
 
-            this.fileGroupsData.fetchAll().subscribe(() => {
+            this.fileGroupsData.fetchAll().subscribe((groups: List<BlobContainer>) => {
```

Comment:
why do you actually need to add this? Doesn't the sdk will create the file group if it doesn't exists?

Reply:
yea i guess i create it as i didn't trust the CLI would actually always create it, and also for validation. Just seemed easier for me to create it and have validation pass.

Reply:
well I think then it goes against the UI which says that the file group will be created if you give a name that is not found